src/styles/index.scss

```diff
@@ -1,3 +1,78 @@
 body {
   margin: 0;
+  min-height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  box-sizing: border-box;
+}
+
+.calendar {
+  padding: 10px;
+  display: flex;
```

Comment:
Consider refactoring repeated `px` values into variables for better maintainability and readability. This will help make your code more efficient and easier to update in the future.